internal/crypto/tls.go

```diff
@@ -54,5 +54,6 @@ func TLSConfigForClient(caConfig, clientConfig *TLSCertificateConfig) (*tls.Conf
 	return &tls.Config{
 		RootCAs:      caPool,
 		Certificates: []tls.Certificate{cert},
+		MinVersion:   tls.VersionTLS12,
```

Comment:
The server uses min TLS1.3: https://github.com/flightctl/flightctl/blob/49a37ef878677399c7aa6ff7475bddf08156c968/internal/crypto/tls.go#L32

Reply:
Fixed